cmake/modules/BuildArrow.cmake

```diff
@@ -140,4 +151,13 @@ function(build_arrow)
   set_target_properties(Arrow::Parquet PROPERTIES
     IMPORTED_LINK_INTERFACE_LANGUAGES "CXX"
     IMPORTED_LOCATION "${parquet_LIBRARY}")
+
+  if(WITH_ARROW_FLIGHT)
+    add_library(Arrow::Flight STATIC IMPORTED)
+    add_dependencies(Arrow::Flight arrow_ext)
+    target_link_libraries(Arrow::Flight INTERFACE Arrow::Arrow "grpc++")
```

Comment:
How do I get linking against Arrow::Flight to also include libprotobuf.so and libgrpc++.so ?